### PR TITLE
Add importation of db from FirstUse Auth

### PR DESCRIPTION
**Pull request by leportella**
As suggested on [this TLJH issue](https://github.com/jupyterhub/the-littlest-jupyterhub/issues/264), we should have a method to import data from FirstUse Authenticator in case a user would like to change authenticators. 

However, Native Auth can return an empty user if the user is not sanitized (has commas or whitespaces) or have option for a password security check. Thus, I added an error to be raise, so the person should know that user won't be able to be created.

### CODE REVIEW

**Comment by leportella** [comment]
@yuvipanda not sure how to do this. Could you give me more details or an similar example, please?

File: nativeauthenticator/nativeauthenticator.py [hunk 1]
```diff
@@ -50,6 +51,18 @@ class NativeAuthenticator(Authenticator):
         default=False,
         help="Asks for email on signup"
     )
+    import_from_firstuse = Bool(
+        config=True,
+        default=False,
+        help="Import users from FirstUse Authenticator database"
+    )
+    dbm_path = Unicode(
```
Comment (yuvipanda) [inline nativeauthenticator/nativeauthenticator.py]: Call this firstuse_dbm_path to make it clear what is in the dbm file?

**Comment by leportella** [comment]
@yuvipanda I added the changes. The tests pass locally, but they don't pass on circleci. I can't imagine what could be wrong. Any ideia?

**Comment by yuvipanda** [comment]
@leportella hmm, maybe you accidentally have a jupyterhub_config.py file that has correct path? Also do we need to manually add the '.db' suffix when deleting? It should be the same as the path passed in to us, right?

**Comment by leportella** [comment]
The thing is... for some reason `dbm` library doesn't accept namefiles with `.db`. 
When I open/create a file called `dbm.db` it actually creates a file called `dbm.db.db`. And if try to open this file, it raises an error. So, the name file on the attribute should **not** have a `.db` in the end... which generates a lot of mess

screen shot 2019-02-26 at 22 00 25

**Comment by leportella** [comment]
@yuvipanda I tried to fix the tests using the full path of the file given by the `Path` but is not working still. Go back to this tomorrow

**Comment by yuvipanda** [comment]
@leportella cool. CircleCI lets you ssh into the build system - https://circleci.com/docs/2.0/ssh-access-jobs/. Helps debug issues like this much easier!

One thing I'd recommend is to try construct the absolute path to the file first, rather than a relative path. Currently, the path to file is relative to 'current directory'. You can and should turn into an absolute path (starting with '/') with something like `os.path.abspath(dbm_path)`. This should also help catch issues with different 'current directory' than what you expect.

**Comment by leportella** [comment]
@yuvipanda found the problem using the Circle CI ssh! 
The problem is the different behavior of the library `dbm` depending on the operational systems. My mac is BDS based, so it adds the `.db` at the end, but the Circle CI system is not, so it doesn't. The problem is only on deleting the file, so I added a verification there :)

**Comment by yuvipanda** [comment]
Great Job, @leportella! \o/

**Pull request merged**